etc/spotlight/sparql_parser.y

```diff
@@ -85,45 +90,61 @@ expr                           {
 
 expr:
 BOOL {
-	/*
-	 * We can't properly handle these in expressions, fortunately this
-	 * is probably only ever used by OS X as sole element in an
-	 * expression i.e. "False" (when Finder window selected our share
-	 * but no search string entered yet). Packet traces showed that OS
-	 * X Spotlight server then returns a failure (ie -1) which is what
-	 * we do here too by calling YYABORT.
-	 */
-	YYABORT;
+    /*
+     * We can't properly handle these in expressions, fortunately this
+     * is probably only ever used by OS X as sole element in an
+     * expression i.e. "False" (when Finder window selected our share
+     * but no search string entered yet). Packet traces showed that OS
+     * X Spotlight server then returns a failure (ie -1) which is what
+     * we do here too by calling YYABORT.
+     */
+    YYABORT;
 }
 | match OR match                 {
     if ($1 == NULL || $3 == NULL)
+    {
         YYABORT;
+    }
+
     if (strcmp($1, $3) != 0)
+    {
         $$ = talloc_asprintf(ssp_slq, "{ %s } UNION { %s }", $1, $3);
-    else
+    } else
+    {
         $$ = talloc_asprintf(ssp_slq, "%s", $1);
+    }
 }
 | match                        {$$ = $1; if ($$ == NULL) YYABORT;}
 | function                     {$$ = $1;}
 | OBRACE expr CBRACE           {$$ = talloc_asprintf(ssp_slq, "%s", $2);}
 | expr AND expr                {
-    if (!ssp_slq->slq_allow_expr) {
+    if (!ssp_slq->slq_allow_expr)
```

Comment:
are we intentionally using a different brace style here than in the rest of the C? (e.g., if we're using 1tbs everywhere else, {s should be on the same line as ifs)

Reply:
good catch; not sure why astyle didn't apply the braces rule consistently

Reply:
manually fixed all instances of if and else